src/ring_theory/dedekind_domain/adic_valuation.lean

```diff
@@ -333,4 +333,71 @@ instance adic_completion.has_lift_t : has_lift_t K (v.adic_completion K) :=
 /-- The ring of integers of `adic_completion`. -/
 def adic_completion_integers : valuation_subring (v.adic_completion K) := valued.v.valuation_subring
 
+variables (R K)
+
+lemma adic_completion.is_integer (x : v.adic_completion K) :
+  x ∈ v.adic_completion_integers K ↔ (valued.v x : ℤₘ₀) ≤ 1 :=
+iff.rfl
+
+section algebra_instances
+
+@[priority 100] instance adic_valued.has_uniform_continuous_const_smul' :
```

Comment:
The parameter `v` does not appear in the target of this instance, `algebra R K`, and is thus underdetermined. For this reason, this should be a `lemma` rather than an `instance`:
```suggestion
lemma adic_valued.has_uniform_continuous_const_smul' :
```